pallets/registrar/src/lib.rs

```diff
@@ -585,6 +596,30 @@ pub mod pallet {
 
             Ok(())
         }
+
+        #[pallet::call_index(8)]
+        #[pallet::weight(T::WeightInfo::set_para_manager())]
+        pub fn set_para_manager(
```

Comment:
Can we by default populate ParaManager with creator at registration? we probably need a migration for that too :(

Reply:
Either that or in `is_para_manager` we compare against the the creator if the paraManager is None